Tasks/MSBuild/task.json

```diff
@@ -6,131 +6,131 @@
     "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=613724)",
```

Comment:
Same deal here. The task.loc.json (generated from the task.json during the build) did not change other than the patch version. That confirms the changes here are safe.